src/e2e-tests/page-objects/smartthings/backRoomTab.page.js

```diff
@@ -0,0 +1,69 @@
+// backRoomTab.page.js
+import Page from '../page';
+import MenuComponent from '../menu.component';
+
+class BackRoomTabPage extends Page {
```

Comment:
BackRoom shall not extend page because it is not a page, it is more a component, set of options

Reply:
Updated.